build/Makefile

```diff
@@ -126,6 +126,7 @@ clean : clean-status
 clean-ruby : clean
 	sudo $(RMDIR) $(BASE_DIR)/intermediate
 	sudo $(RMDIR) $(RUBY_DIR)/.ext
+	sudo $(RMDIR) /usr/local/ruby-2.2.0*
```

Comment:
@jeffaco 
Moved local ruby for testing to be built under intermediate. and having $(RMDIR) $(BASE_DIR)/intermediate handle clean up of local ruby automatically .